<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeDegradeurDialog</class>
 <widget class="QWidget" name="ChangeDegradeurDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change degradeur(s)</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="formLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="FormLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="degradeur1Label">
      <property name="text">
       <string>Degradeur 1 (mm plexi)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="degradeur1ComboBox">
      <property name="styleSheet">
       <string notr="true">QComboBox 
{
background: white;
}

QComboBox QAbstractItemView 
{
background-color: white;
selection-color: dark-grey;
}</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="degradeur2Label">
      <property name="text">
       <string>Degradeur 2 (mm plexi)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="degradeur2ComboBox">
      <property name="styleSheet">
       <string notr="true">QComboBox 
{
background: white;
}

QComboBox QAbstractItemView 
{
background-color: white;
selection-color: dark-grey;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancelPushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="updatePushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Effectuer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
